src/window.rs

```diff
@@ -38,18 +38,6 @@ static WINDOW_ICON: &'static [u8] = include_bytes!("../assets/windows/alacritty.
 /// In X11, this the default value for the `WM_NAME` property.
 pub const DEFAULT_TITLE: &str = "Alacritty";
 
-/// Default text for general window class, X11 specific.
-///
-/// In X11, this is the default value for the `WM_CLASS` property. The
-/// second value of `WM_CLASS` is **never** changed to anything but
-/// the default value.
-///
-/// ```ignore
-/// $ xprop | grep WM_CLASS
-/// WM_CLASS(STRING) = "Alacritty", "Alacritty"
-/// ```
-pub const DEFAULT_CLASS: &str = "Alacritty";
```

Comment:
Do we really want to remove all support for separate classes and titles under X11? I'm personally pretty indifferent, but I don't really know all the cases we might want to use window classes?

Given that Wayland doesn't seem to have window classes (this is correct right?), I can understand this removal.

Reply:
We aren't. The functionality hasn't been removed. The only thing that has been removed is a separation between two constants which should always be the same anyways.

The `DEFAULT_CLASS` constant was actually only used in `src/cli.rs`, even the default class in the `window.rs` was using `DEFAULT_TITLE`. If `_TITLE` is the best name or not could be discussed here, maybe `DEFAULT_NAME` would be better. But I don't think there's any point in storing the same thing twice.

Reply:
Sure, I agree with that. I think I agree that `DEFAULT_NAME` is a better constant name here, if you want to update that.